### PR TITLE
[docs] Documentation fixes for built-in functions

### PR DESCRIPTION
**Pull request by DaVukovic**
## Description

This PR fixes some layout problems in the  docs as some commas haven't been escaped. 

I also changed `Skin.ToggleDebug` to `ToggleDebug` for the docs at `ApplicationBuitins.cpp` because the specific code doesn't toggle the Skin-debugmode but toggles the debuglog. Also `Skin.ToggleDebug` is specified at `SkinBuitlins.cpp`.

`InstallFromZip` wasn't documented at all, as well es tthe PlayerControl option for `TempoUp` or `TempoDown`

 
## Motivation and Context

Keeping the documentation up to date.

## How Has This Been Tested?

I compiled the doxygen HTML files to check the changes and also checked if the specific built-in functions are working as expected via `kodi-send -a "<some_built_in>"`.

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [x] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [x] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [x] I have updated the documentation accordingly
- [x] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: xbmc/interfaces/builtins/PlayerBuiltins.cpp [hunk 1]
```diff
@@ -533,6 +533,8 @@ static int Seek(const std::vector<std::string>& params)
 ///     as parameter.
 ///     @param[in] control               Control to execute.
 ///     @param[in] param                 "notify" to notify user (optional\, certain controls).
+///
+///     @note TempoUp or TempoDown only works if "Sync playback to display" is enabled.
```
Comment (enen92) [inline xbmc/interfaces/builtins/PlayerBuiltins.cpp]: (Added in v18)

**Comment by AlwinEsch** [comment]
Looks good, Thanks a lot!

**Comment by DaVukovic** [comment]
Added some missing controls and didn't mentioned "record" anymore as that's not handled by `PlayerBuiltins.cpp`. "Record" is handled at `PVRBuiltins.cpp` via `PVR.ToggleRecordPlayingChannel`.

@enen92 
Added a complete new table for better readability and an additional field when that specific control was added. I guess that's fine?!

@AlwinEsch 
Could you please review? Thx in advance.

**Comment by ronie** [comment]
> the specific code doesn't toggle the Skin-debugmode but toggles the debuglog.

nope, it **does** toggle the skin debug mode.

**Comment by DaVukovic** [comment]
@ronie

I'm talking about that code:

```
static int ToggleDebug(const std::vector<std::string>& params)
{
  bool debug = CServiceBroker::GetSettings().GetBool(CSettings::SETTING_DEBUG_SHOWLOGINFO);
  CServiceBroker::GetSettings().SetBool(CSettings::SETTING_DEBUG_SHOWLOGINFO, !debug);
  g_advancedSettings.SetDebugMode(!debug);

  return 0;
}
```

That does toggle the Skin-Debug mode?

File: xbmc/interfaces/builtins/PlayerBuiltins.cpp [hunk 2]
```diff
@@ -519,20 +519,43 @@ static int Seek(const std::vector<std::string>& params)
...
+///     | RepeatOff               | Sets 'Repeat' to 'off'                 | Sets 'Repeat' to 'off'      |             |
+///     | Partymode(music) **     | none                                   | Toggles music partymode     |             |
+///     | Partymode(video) **     | Toggles video partymode                | none                        |             |
+///     | Partymode(path to .xsp) | Partymode for *.xsp-file               | Partymode for *.xsp-file    |             |
+///     | ShowVideoMenu           | Shows the DVD/BR menu if available     | none                        |             |
+///     <br>
+///     '*' = These take an optional second parameter\, Notify\, that notifies the user of the new state<br>
+///     '**' = If no parameter is given\, Partymode will default to music
+///     <br>
 ///     @param[in] control               Control to execute.
```
Comment (enen92) [inline xbmc/interfaces/builtins/PlayerBuiltins.cpp]: either change control to command or command to control (see built-in definition at the top)

File: xbmc/interfaces/builtins/PlayerBuiltins.cpp [hunk 3]
```diff
@@ -519,20 +519,43 @@ static int Seek(const std::vector<std::string>& params)
...
+///     | RepeatOne               | Repeats a single video                 | Repeats a single track      |             |
+///     | RepeatAll               | Repeat all videos in a list            | Repeats all tracks in a list|             |
+///     | RepeatOff               | Sets 'Repeat' to 'off'                 | Sets 'Repeat' to 'off'      |             |
+///     | Partymode(music) **     | none                                   | Toggles music partymode     |             |
+///     | Partymode(video) **     | Toggles video partymode                | none                        |             |
+///     | Partymode(path to .xsp) | Partymode for *.xsp-file               | Partymode for *.xsp-file    |             |
+///     | ShowVideoMenu           | Shows the DVD/BR menu if available     | none                        |             |
+///     <br>
+///     '*' = These take an optional second parameter\, Notify\, that notifies the user of the new state<br>
+///     '**' = If no parameter is given\, Partymode will default to music
```
Comment (enen92) [inline xbmc/interfaces/builtins/PlayerBuiltins.cpp]: maybe change parameter to something else as parameters may be confused with the built-in arguments (i.e. we have `BuiltinFunction(control, param)` and in this case we have `BultinFunction(Control(arg), param)`

File: xbmc/interfaces/builtins/PlayerBuiltins.cpp [hunk 4]
```diff
@@ -519,20 +519,43 @@ static int Seek(const std::vector<std::string>& params)
...
+///     | Repeat *                | Cycles through repeat modes            | Cycles through repeat modes |             |
+///     | RepeatOne               | Repeats a single video                 | Repeats a single track      |             |
+///     | RepeatAll               | Repeat all videos in a list            | Repeats all tracks in a list|             |
+///     | RepeatOff               | Sets 'Repeat' to 'off'                 | Sets 'Repeat' to 'off'      |             |
+///     | Partymode(music) **     | none                                   | Toggles music partymode     |             |
+///     | Partymode(video) **     | Toggles video partymode                | none                        |             |
+///     | Partymode(path to .xsp) | Partymode for *.xsp-file               | Partymode for *.xsp-file    |             |
+///     | ShowVideoMenu           | Shows the DVD/BR menu if available     | none                        |             |
+///     <br>
+///     '*' = These take an optional second parameter\, Notify\, that notifies the user of the new state<br>
```
Comment (enen92) [inline xbmc/interfaces/builtins/PlayerBuiltins.cpp]: maybe the case doesn't matter at all but `Notify` is lowercase below, so please keep one style for consistency. Also, the wording may lead to confusion: it's hard to understand if notify is a parameter for the command or the built-in itself (it is likely the correct one). Maybe change wording to something like "For these commands, the `PlayerControl` built-in function can make use of the notify parameter...."

**Comment by ronie** [comment]
please ignore what i said.. sorry for the noise ;-)

**Comment by DaVukovic** [comment]
No probs ;)

**Review by enen92 — APPROVED** [review]

**Pull request merged**